lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -105,6 +105,15 @@ module.exports.prototype = {
                     return;
                 }
 
+                // Check if this block is used in call expression
+                if (parentNode.parentNode.type === 'CallExpression') {
```

Comment:
@TheSavior 's solution is correct in that it won't barf on top level blocks. However, I don't think we need to add any options.